crates/rpc/rpc/src/eth/api/call.rs

```diff
@@ -35,7 +35,8 @@ use tracing::trace;
 
 // Gas per transaction not creating a contract.
 const MIN_TRANSACTION_GAS: u64 = 21_000u64;
-const MIN_CREATE_GAS: u64 = 53_000u64;
+// Allowed error ratio for gas estimation
+const ESTIMATE_GAS_ERROR_RATIO: f64 = 0.015;
```

Comment:
where is this coming from?
is this 1 - 64/63 ?

can we do `1. - 64. / 63.` here?
plus one line doc with geth reference?

Reply:
It's `1. - 64. / 63.` but truncated to 3 decimals. Do you want the full explicit computation? `f64::trunc((1. - 63. / 64.) * 1000.) / 1000.`?

Reply:
then 0.015 is fine